airflow/cli/commands/dag_command.py

```diff
@@ -300,15 +300,17 @@ def dag_list_dag_runs(args, dag=None):
 
     dagbag = DagBag()
 
-    if args.dag_id not in dagbag.dags:
+    if args.dag_id is not None and args.dag_id not in dagbag.dags:
```

Comment:
Does that fix a bug? When would a `dag_id` be `None`? The `dag_id` is required, isn't it?

Reply:
No, dag_id is optional parameter in this request. It is requirement of this task. See third point of the task.

Reply:
You are right. 👍